Source/WebCore/page/Page.cpp

```diff
@@ -262,6 +262,14 @@ static constexpr OptionSet<ActivityState::Flag> pageInitialActivityState()
     return { ActivityState::IsVisible, ActivityState::IsInWindow };
 }
 
+#if HAVE(WORK_INTERVAL_API)
+static WTF::WorkgroupInterval& workgroupRendering()
```

Comment:
I'd still prefer to see this outside of Page.cpp.

Reply:
Yes, I agrees. Will fix, likely in a new PR.

Thanks!